packages/core/http/core-http-router-server-internal/src/versioned_router/core_versioned_route.ts

```diff
@@ -138,6 +146,9 @@ export class CoreVersionedRoute implements VersionedRoute {
       validation?.request &&
       Boolean(validation.request.body || validation.request.params || validation.request.query)
     ) {
+      if (this.enableQueryVersion) {
+        redactQueryVersion(mutableCoreKibanaRequest);
+      }
```

Comment:
NIT: I had to check the implementation of `redactQueryVersion` to see what it was doing. It's more `remove` than `redact` actually.